documentation/1.0/content/setup/install.md

```diff
@@ -9,13 +9,14 @@ description: "Install the WKT UI application and check for updates."
 1. Download the latest WebLogic Kubernetes Toolkit UI (WKT UI) application installers from the [GitHub Releases section](https://github.com/oracle/weblogic-toolkit-ui/releases) of this repository.
 2. Simply run the appropriate installer for your operating system.
 
-**NOTE**: When installing on Linux, to get _all_ the dependencies and installed in the correct order, use a command similar to the following:
+**NOTE**: On Linux, to get _all_ the dependencies and have them installed in the correct order, you need to use the package manager to install the `rpm` or `deb` file.
+
+- On Oracle/RedHat/CentOS (and some others), use either `yum` or `dnf`; for example:
 ```
 sudo yum localinstall wktui-1.0.0.x86_64.rpm
 ```
 
-- On Oracle/RedHat/CentOS (and some others), use either `yum` or `dnf`
-- On Ubuntu, run:
+- On Ubuntu, use `apt-get`; for example:
```

Comment:
@robertpatrick @jshum2479 I know that there might still be some discussion going on here about this, but I wonder if we need another sentence/explanation between stating, "On Ubuntu, use `apt-get`; for example:" and "sudo dpkg -i wktui_1.0.0_amd64.deb," something to explain why we are showing this command `sudo dpkg -i wktui_1.0.0_amd64.deb`.

Reply:
@jshum2479 @rosemarymarano It's not clear to me that we need to show the usage of `dpkg -i`.  Unless there is some problem that the `dpkg -i` is solving that prevents the `apt-get` command from working, I would opt to remove the `dpkg -i` section completely and just show the `apt-get` command.  

If there is a problem it is trying to solve, then I agree that we need an explanation of the problem, what in the `dpkg -i` output the user should look for, and how the follow-on `apt-get` command solves the problem.  However, we probably need the basic `apt-get` command prior to discussing a potential problem that might be encountered and how to address it.  Can the two of you please work together to find the right organization of this section?